src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc

```diff
@@ -256,6 +197,54 @@ void AlertManeuverRequest::on_event(const event_engine::Event& event) {
                &(message[strings::msg_params]));
 }
 
+bool AlertManeuverRequest::CalculateResultResultCode(
+    mobile_apis::Result::eType& result_code, std::string& return_info) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  using namespace helpers;
+  bool is_tts_ok;
+  bool is_no_navi_error;
+  hmi_apis::Common_Result::eType tts_result;
+  hmi_apis::Common_Result::eType navi_result;
+
+  navi_result =
+      MessageHelper::MobileToHMIResult(navi_alert_maneuver_result_code_);
+  is_no_navi_error = Compare<hmi_apis::Common_Result::eType, EQ, ONE>(
+      navi_result,
+      hmi_apis::Common_Result::SUCCESS,
+      hmi_apis::Common_Result::WARNINGS);
+
+  tts_result = MessageHelper::MobileToHMIResult(tts_speak_result_code_);
+
+  is_tts_ok = Compare<hmi_apis::Common_Result::eType, EQ, ONE>(
+      tts_result,
+      hmi_apis::Common_Result::SUCCESS,
+      hmi_apis::Common_Result::UNSUPPORTED_RESOURCE,
+      hmi_apis::Common_Result::WARNINGS,
+      hmi_apis::Common_Result::INVALID_ENUM);
+
+  bool result = is_tts_ok && is_no_navi_error;
+  result =
+      result || (hmi_apis::Common_Result::SUCCESS == tts_result &&
+                 hmi_apis::Common_Result::UNSUPPORTED_RESOURCE == navi_result);
```

Comment:
@dtrunov 
I think you may add UNSUPPORTED_RESOURCE to is_no_navi_error calculation and remove this check.